editor/state.js

```diff
@@ -89,59 +89,72 @@ export const editor = combineUndoableReducers( {
 					};
 				}, omit( state, action.uids ) );
 
-			case 'REMOVE_BLOCK':
-				return omit( state, action.uid );
+			case 'REMOVE_BLOCKS':
+				return omit( state, action.uids );
 		}
 
 		return state;
 	},
 
 	blockOrder( state = [], action ) {
-		let index;
-		let swappedUid;
 		switch ( action.type ) {
 			case 'RESET_BLOCKS':
 				return action.blocks.map( ( { uid } ) => uid );
 
-			case 'INSERT_BLOCK':
+			case 'INSERT_BLOCK': {
```

Comment:
Yeah I'd rather scope the `const` declarations to one case instead of the whole reducer. You can use `{}` anywhere you want AFAIK.

```js
/* ... */

{
    let something = 1;
}

/* ... */
```